### PR TITLE
readme: update book.* links

### PR DESCRIPTION
**Pull request by OverOrion**
This PR updates the README links, let me know if the `Build and Run` section should point to a specific chapter like it used to.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # integritee-node
 
-This repository belongs to the [Integritee project](https://book.integritee.network).
+This repository belongs to the [Integritee project](https://integritee.network).
 
 A substrate-based node that maintains a registry of remote attested integritee-service enclaves. The node also acts as a proxy for encrypted requests which are forwarded to the integritee-service.
 
 ## Build and Run
-Please see our [Integritee Book](https://book.integritee.network/howto_node.html) to learn how to build and run this.
+Please see our [Integritee Documentation](https://docs.integritee.network) to learn how to build and run this.
```
Comment (clangenb) [inline README.md]: Hmm, I just noticed that the Integritee book does not have a section about the standalone node. I think we can just inline the compile docs here.

1. See the substrate install docs to install the preliminaries: https://docs.substrate.io/install/
2. Build the node with:
   * `cargo build --release --features "skip-extrinsic-filtering"` or
   * `cargo build --release --features "skip-extrinsic-filtering skip-ias-check"` (if you don't have SGXhardware aka can't perform a remote attestation)

**Review by clangenb — COMMENTED** [review]
As always, thanks for the maintenance! I have only one remark. :)

File: README.md (hunk 1 above)
Comment (OverOrion) [reply README.md]: Inlined it, should we add a link to explain what Remote Attestation is?

Comment (clangenb) [reply README.md]: No, I guess this is fine. The link to the Integritee docs is just a few lines above. Thanks for fixing it!

**Review by clangenb — COMMENTED** [review]
Thanks, looks good to me now!

**Review by clangenb — APPROVED** [review]

**Pull request merged**